src/coord/src/coord.rs

```diff
@@ -5741,6 +5877,22 @@ mod timeline {
             }
         }
 
+        /// Peek the current value of the timestamp.
+        ///
+        /// No operations should be assigned to the timestamp returned by this function. The
+        /// timestamp returned should only be used to compare the progress of the `TimestampOracle`
+        /// against some external source of time.
+        ///
+        /// Subsequent values of `self.read_ts()` and `self.write_ts()` will be greater or equal to
+        /// this timestamp.
+        ///
+        /// NOTE: This can be removed once DDL is included in group commits.
+        pub fn peek_ts(&self) -> T {
+            match &self.state {
+                TimestampOracleState::Writing(ts) | TimestampOracleState::Reading(ts) => ts.clone(),
+            }
+        }
+
```

Comment:
@frankmcsherry This is the change to the `TimestampOracle`, it's used to determine how long to wait in between writes.